Resources/Prototypes/Entities/Structures/Power/Generation/Tesla/energyball.yml

```
@@ -0,0 +1,115 @@
+- type: entity
+  id: BaseEnergyBall
+  abstract: true
+  components:
+  - type: Clickable
+  - type: Physics
+    bodyType: Dynamic
+    bodyStatus: InAir
```

```suggestion
  - type: Physics
    bodyType: Dynamic
    bodyStatus: InAir
    sleepingAllowed: false
```

This fixes the BB disappearing.